addons/respawn/CfgVehicles.hpp

```diff
@@ -0,0 +1,79 @@
+class CfgVehicles {
+    class Logic;
+    class Module_F: Logic {
+        class AttributesBase {
+            class Combo;
+            class EGVAR(common,codeSmall);
+            class ModuleDescription;
+        };
+
+        class ModuleDescription {
+          class EmptyDetector;
+          class Condition;
+        };
+    };
+
+    class GVAR(moduleRespawn): Module_F {
+        scope = 2;
+
+        displayName = CSTRING(ModuleRespawn_DisplayName);
+        category = QUOTE(PREFIX);
+        icon = "\a3\Modules_f\data\iconRespawn_ca.paa";
+        portrait = "\a3\Modules_f\data\portraitRespawn_ca.paa";
+
+        function = QFUNC(moduleRespawn);
+        isGlobal = 0;
+        isTriggerActivated = 1;
+        isDisposable = 1;
+
+        class Attributes: AttributesBase {
+
+            class GVAR(applyTo): Combo {
+                displayName = CSTRING(ModuleRespawn_Attribute_ApplyTo_DisplayName);
+
+                property = QGVAR(applyTo);
+                typeName = "NUMBER";
+
+                class Values {
+                    class EAST {
+                        name = "$STR_A3_CfgGroups_East0";
+                        value = 0;
+                    };
+                    class WEST {
+                        name = "$STR_A3_CfgGroups_West0";
+                        value = 1;
+                        default = 1;
+                    };
+                    class INDEP {
+                        name = "$STR_A3_CfgGroups_Indep0";
+                        value = 2;
+                    };
```

Comment:
What about civilian side respawn? It could be useful on some missions. I believe this shouldn't be a problem as long as it works on players only.